docs/detections/value-list-exceptions.asciidoc

```diff
@@ -17,11 +17,15 @@ TIP: You can also use a value list as the <<indicator-value-lists,indicator matc
 [float]
 [[create-value-lists]]
 === Create value lists
-When creating a value list for a rule exception, be mindful of the list's size and data type. In general, all rule types support value list exceptions, but there are some limitations for especially large lists or certain data types. The following value list types can _only_ be used with custom query, machine learning, and indicator match rule types:
+When you create a value list for a rule exception, be mindful of the list's size and data type. All rule types support value list exceptions, but extremely large lists or certain data types have limitations. 
+
+Custom query, machine learning, and indicator match rules support the following value list types and sizes:
+
+* **Keywords** or **IP addresses** list types with more than 65,536 values
+* **IP ranges** list type with over 200 dash notation values (for example, `127.0.0.1-127.0.0.4` is one value) or more than 65,536 CIDR notation values
 
 * Keyword or IP address lists with more than 65,536 values
 * IP range lists with more than 200 dash notation values (for example, `127.0.0.1-127.0.0.4` is one value) or more than 65,536 CIDR notation values
```

Comment:
Looks like this duplicated the bullet list instead of editing it.

<img width="875" alt="image" src="https://github.com/elastic/security-docs/assets/87339667/7a5cda78-35a3-4d5b-b0f8-06916a9bea15">

Reply:
Resolved in [cf38995](https://github.com/elastic/security-docs/pull/5001/commits/cf389958190abcec5ff23a1ff60005239fc12943).